src/plugins/GUITestBase/src/tests/common_scenarios/Assembling/extract_consensus/GTTestsAssemblyExtractConsensus.cpp

```diff
@@ -0,0 +1,327 @@
+            CHECK_SET_ERR(cbAlgo != nullptr, "Algorithm widget is nullptr");
+            GTComboBox::selectItemByIndex(os, cbAlgo, static_cast<int>(algo));
+        }
+        {
+            auto *const cbKeepGaps = GTWidget::findExactWidget<QComboBox *>(os, "Keep gaps widget", dialog);
+            CHECK_SET_ERR(cbKeepGaps != nullptr, "Keep gaps widget is nullptr");
+            GTComboBox::selectItemByIndex(os, cbKeepGaps, static_cast<int>(keepGaps));
+        }
+        {
+            auto *const urlleOut = GTWidget::findExactWidget<QLineEdit *>(os, "Output file widget", dialog);
+            CHECK_SET_ERR(urlleOut != nullptr, "Output file widget is nullptr");
+            GTLineEdit::setText(os, urlleOut, outFile);
+        }
+        GTUtilsWizard::clickButton(os, GTUtilsWizard::Run);
+    }
+};
+
+using Algo = ExtractConsensusWizardScenario::Algorithm;
+
+GUI_TEST_CLASS_DEFINITION(sorted_bam_test_0001) {
+    const GTLogTracer lt;
+    const QString path = dataDir + "samples/Assembly/chrM.sorted.bam";
+
+    //  1. Select "Tools->NGS data analysis->Extract consensus from assemblies..."
+    //  2. Set "samples/Assembly/chrM.sorted.bam" as an input
+    //  3. Click "Run"
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(
+        os, "Extract Consensus Wizard", new ExtractConsensusWizardScenario(QStringList() << path)
+    ));
+    GTMenu::clickMainMenuItem(os, QStringList() << "Tools"
+                                                << "NGS data analysis"
+                                                << "Extract consensus from assemblies...");
+
+    //  4. Wait for workflow finished
+    GTUtilsTaskTreeView::waitTaskFinished(os, 100000);
+    //  Expected state: There should be no errors in the log
+    CHECK_SET_ERR(!lt.hasErrors(), "Errors in log: " + lt.getJoinedErrorString());
+    //  There should be no notifications in the dashboard
+    CHECK_SET_ERR(!GTUtilsDashboard::hasNotifications(os), "Unexpected notification");
+
+    //  5. Return to workflow and call the Extract consensus wizard
+    GTUtilsWorkflowDesigner::returnToWorkflow(os);
+    //  6. Fill in the dialog :
+    //      {Algorithm} SAMtools
+    //      {Keep gaps} False
+    //      {Output file}
+    //     And click "Run"
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(
+        os,
+        "Extract Consensus Wizard",
+        new ExtractConsensusWizardScenario(QStringList() << path, Algo::SAMTOOLS, false, "")
+    ));
+    GTToolbar::clickButtonByTooltipOnToolbar(os, MWTOOLBAR_ACTIVEMDI, "Show wizard");
+
+    //  7. Wait for workflow finished
+    GTUtilsTaskTreeView::waitTaskFinished(os, 100000);
+    //  Expected state: There should be no errors in the log
+    CHECK_SET_ERR(!lt.hasErrors(), "Errors in log: " + lt.getJoinedErrorString());
+    //  There should be no notifications in the dashboard
+    CHECK_SET_ERR(!GTUtilsDashboard::hasNotifications(os), "Unexpected notification");
```

Comment:
nit: include the notification text in the assertion message to simplify analysis when the error will happen.